CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Features
+
+- [Structured Logging] Add `SentryLog` Models ([#5441](https://github.com/getsentry/sentry-cocoa/pull/5441))
```

Comment:
`m`: If I'm not mistaken, you can't use SentryLogs yet. So I would ditch the changelog.